packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-credit-button-strategy.ts

```diff
@@ -40,17 +49,32 @@ export default class PaypalCommerceCreditButtonStrategy implements CheckoutButto
         const currencyCode = state.cart.getCartOrThrow().currency.code;
         const paymentMethod = state.paymentMethods.getPaymentMethodOrThrow(methodId);
         this._paypalCommerceSdk = await this._paypalScriptLoader.getPayPalSDK(paymentMethod, currencyCode, initializesOnCheckoutPage);
-
-        this._renderButton(containerId, methodId, initializesOnCheckoutPage, style);
+        const { initializationData } = paymentMethod;
+        const { isHostedCheckoutEnabled } = initializationData;
+        this._renderButton(containerId, methodId, isHostedCheckoutEnabled, initializesOnCheckoutPage, style);
```

Comment:
To avoid passing extra props to `this._renderButton` method, you can get `isHostedCheckoutEnabled` from payment method inside `this._renderButton`

Reply:
fixed